CrosswordMe.py

```diff
@@ -1,17 +1,46 @@
+from random import randint
+
 #input
 
-words = []
+words = ["one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten"]
 user_input = ""
+print(len(words))
 
-while 1 == 1:
+"""while 1 == 1:
     user_input = input("Word: ")
     if user_input.lower() == "done":
         break
-    words.append(user_input)
+    words.append(user_input)"""
 
 # creating the puzzle
+## horizontals
+line = randint(0,9) #First line to write in
+lines_used = [] #Empty array which contains lines which have been written in already (can probably be removed when its possible to write multiple words in one line)
+
+puzzle = [["$" for x in range(10)] for y in range(10)] #Creates 10x10 array filled with "$"
+
+for word in range(0, len(words)): #loops through every word in the words list
+
+
+
+    word_position = 0 #resets the word_position. this is needed beacuse word_position counts through the length of every word being written
+
+    start_position = randint(0, 10-len(words[word])) #int for start_position
+    end_position = start_position+len(words[word]) #int for end_position
+
+    for array_position in range(start_position, end_position): #Loops through position within the array
+
+        puzzle[line][array_position] = words[word][word_position] #Writes the word at random position of the array
+        word_position += 1 #Fake loop for the word position
 
-puzzle = [[[0 for x in range(10)] for y in range(10)] ]
+    lines_used.append(line) #appends used lines to used lines list
+    line = randint(0,9) #Choosees new line
+    while line in lines_used and word != 9: #keeps creating a new line as long as the line has been used alreardy and its not the last word
```

Comment:
this is going to leave you hanging in an infinite loop if you aren't careful. Like when you change the number of words.

Also it is unnecessary to calculate  random numbers and then check if they are used.
You could either generate a random line out of the set of not-used lines.
Or just generate a random order of lines up front and then loop over them:
```
PS C:\Users\fabian> python
Python 3.6.1 (v3.6.1:69c0db5, Mar 21 2017, 18:41:36) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from random import sample
>>> words = ["correct", "horse", "battery", "staple"]
>>> sample(words, len(words))
['correct', 'battery', 'staple', 'horse']
>>> sample(words, len(words))
['staple', 'battery', 'correct', 'horse']
>>> sample(words, len(words))
['horse', 'correct', 'battery', 'staple']
>>>
```